conans/test/utils/tools.py

```diff
@@ -1321,6 +1195,132 @@ def __repr__(self):
         return "\n".join(ret)
 
 
+class TurboTestClient(TestClient):
```

Comment:
Has something changed here? Please do it in another PR, clarifying if it is only a refactor or there are changes in it.

Reply:
As far as I know, these changes were already introduced here: https://github.com/conan-io/conan/pull/5932